### DISCUSSION TITLE
Error loading dynamically imported module in viem documentation: signMessage example fails

### DISCUSSION THREAD

**Discussion by Aymsep** [wevm/viem#2255] [General]
I encountered an issue in the viem documentation while trying to fetch account information. The example provided in the [signMessage](https://viem.sh/assets/signMessage-l97qrV-f.js) section does not work as expected. It throws the following error:

**Error loading dynamically imported module: https://viem.sh/assets/signMessage-l97qrV-f.js**

**Comment by tmm** [comment]
Sounds like an issue with your bundler/build set up. Need to provide more info if anyone is going to take a look at this.
